### ISSUE TITLE
Some OuterLoop tests fail when built with explicit configuration

### ISSUE THREAD

**Issue by roncain** [dotnet/wcf#731]
If you build and run OuterLoop tests by explicitly setting $(Configuration), 3 tests fail.  More importantly, it means the product builds for the wrong target framework if the configuration is specified on the command line.

Command to run the tests is:
```
build.cmd /p:Configuration=Debug /p:WithCategories=OuterLoop
```
The 3 tests that fail (apparently thinking they are for the wrong target framework) are:

Tcp_ClientCredentialTypeTests.SameBinding_DefaultSettings_EchoString [FAIL] (TaskId:6796)
Tcp_ClientCredentialTypeTests.SameBinding_SecurityModeTransport_EchoString [FAIL] (TaskId:6796)
HttpsTests.ServerCertificateValidation_EchoString [FAIL] (TaskId:6796)

**Comment by StephenBonikowsky** [comment]
Fixed by PR #744

**Comment by StephenBonikowsky** [comment]
Clarification
These tests will still fail if "Debug" or "Retail" are passed-in to the build script as the Configuration. This is due to the convoluted logic in the build process.
While we can find the exact place this happens it is debatable whether this is worth the time to do as WCF Core is not OS neutral, it must be built either against WIndows or Linus (Unix).

So this bug is not fixed, just avoided by changing Jenkins to not pass-in just "Debug" or "Retail".
